package.json

```diff
@@ -52,12 +52,12 @@
     "setup": "npm run module && npm run virtualenv && npm install && npm run config",
     "watch": "grunt watch",
     "build": "grunt",
-    "lint": "npm run lintJS && npm run lintCSS",
-    "lintJS": "eslint ./Gruntfile.js ./tests ./grunt-tasks ./webcompat/static/js/lib && grunt travis-lint",
-    "lintCSS": "stylelint ./webcompat/static/css/development/**/*.css",
-    "fix": "npm run fixJS && npm run fixCSS",
-    "fixJS": "eslint --fix ./Gruntfile.js ./tests ./grunt-tasks ./webcompat/static/js/lib",
-    "fixCSS": "stylelint ./webcompat/static/css/development/**/*.css --fix",
+    "lint": "npm run lint:JS && npm run lint:CSS",
+    "lint:JS": "eslint ./Gruntfile.js ./tests ./grunt-tasks ./webcompat/static/js/lib && grunt travis-lint",
+    "lint:CSS": "stylelint ./webcompat/static/css/development/**/*.css",
+    "fix": "npm run fix:JS && npm run fix:CSS",
```

Comment:
@magsout Would you mind changing `fix:CSS / fix:JS` to `lint:fix:CSS / lint:fix:JS`?
Could add more valuable context. What do you think? 

(Sorry for not suggesting it earlier. Just saw it.).